loaders/src/glTF/babylon.glTFFileLoader.ts

```diff
@@ -1179,6 +1179,8 @@ module BABYLON {
                 rootUrl: rootUrl,
 
                 importOnlyMeshes: importOnlyMeshes,
+
+                defaultMaterial: undefined

```

Comment:
super minor, but you should probably be consistent here and use null... make sure to change the check to === null also